### PR TITLE
Ekco 0.11.0 internal load balancer

### PR DESCRIPTION
**Pull request by areed**

### CODE REVIEW

File: src/markdown-pages/add-ons/ekco.md [hunk 1]
```diff
@@ -113,6 +114,12 @@ To avoid race conditions, manually run the ekco-reboot service's shutdown script
 /opt/ekco/shutdown.sh
 ```
 
+### Internal Load Balancer
+
+Ekco 0.11.0+ can maintain an internal load balancer forwarding all traffic from host port 6444 to one of the Kubernetes API server pods.
```
Comment (emosbaugh) [inline src/markdown-pages/add-ons/ekco.md:119]: We should be consistent with case. Are we using EKCO or ekco?

**Review by emosbaugh — APPROVED** [review]

File: src/markdown-pages/add-ons/ekco.md (hunk 1 above)
Comment (areed) [reply src/markdown-pages/add-ons/ekco.md]: EKCO

File: src/markdown-pages/install-with-kurl/index.md [hunk 2]
```diff
@@ -20,26 +20,35 @@ kURL installers support a handful of end-user defined advanced options that can
...
 ```
 
-If the cluster has remote nodes, the install script will print out a command that must be run on every node to update clients to use the new load balancer address:
-
-```bash
-curl https://kurl.sh/latest/tasks.sh | sudo bash -s set-kubeconfig-server https://k8slb.somebigbank.com:6443
-```
-
 To change the load balancer of an existing HA cluster, re-run the install script with the new load balancer address.
 This requires that the [ekco add-on](/docs/add-ons/ekco) is enabled with version 0.6.0+.
```
Comment (ajp-io) [inline src/markdown-pages/install-with-kurl/index.md:53]: So you need EKCO 0.11.0+ to use the internal load balancer, but the new `load-balancer-address` flag is respected back to version 0.6.0?

**Review by ajp-io — APPROVED** [review]

File: src/markdown-pages/install-with-kurl/index.md (hunk 2 above)
Comment (areed) [reply src/markdown-pages/install-with-kurl/index.md:53]: Yes, you need EKCO 0.11.0+ to use the internal load balancer. The `load-balancer-address` flag has always been available in kURL, but it wasn't until EKCO 0.6.0 you could change the load balancer address of an existing installation.

**Pull request merged**